metamorph

```diff
@@ -155,11 +155,13 @@ def run(sub_args):
         sub_args,
         config = config
     )
-
     config['bindpaths'] = bindpaths
-    # config['coassembly'] = sub_args.coa
     config['coassembly'] = False
 
+    # Step 4b. Setup assembly mode
+    #   modes: 0 - megahit + metaspades assembly
+    config["options"]["assembler_mode"] = "0" if sub_args.megahit_only else "1"
```

Comment:
The logic here is opposite of what I would expect for a boolean flag. I see in the Snakefile you accounted for that, but would it be worth just sticking to booleans, or "1 for True" and "0 for false" to avoid ambiguity?